<launch>
        # Node
        <node name="drivenet_node" pkg="drivenet" type="drivenet_node"  output="screen">
                <param name="truck_id" type="int" value="1" />
                <param name="standard_fps" type="int" value="0" />
                <param name="display" type="int" value="0" />
                <param name="input_resize" type="int" value="1" />
                <param name="imgResult_publish" type="bool" value="0" />
        </node>

</launch>

